client/webpack.config.js

```diff
@@ -11,7 +11,7 @@ const devServerPort = 8080;
 
 const config = {
   entry: {
-    coursemology: ['babel-polyfill', './app/index'],
+    coursemology: ['babel-polyfill', './app/index', './app/lib/moment-timezone'],
```

Comment:
Does it need to be a new entry?

Reply:
OK, I think it needs to be an entry. Found an article here which explains the entry: https://medium.com/@rajaraodv/webpack-the-confusing-parts-58712f8fcad9